python/lsst/pipe/tasks/finalizeCharacterization.py

```diff
@@ -613,6 +688,17 @@ def compute_psf_and_ap_corr_map(self, visit, detector, exposure, src, isolated_s
         measured_src : `lsst.afw.table.SourceCatalog`
             Updated source catalog with measurements, flags and aperture corrections.
         """
+        if self.config.remove_initial_photo_calib:
+            if initial_photo_calib is None:
+                self.log.warning("No initial photo calib found for visit %d, detector %d", visit, detector)
+                return None, None, None
+            if not initial_photo_calib._isConstant:
+                # TODO DM-46720: remove this limitation and usage of private (why?!) property.
```

Comment:
Do we want to leave `why?!` on `main`? That seems like a discussion to be had outside of the scope of this ticket.